industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,6 @@
+# GSK Industry Benchmark 
+Prepared by Alexander Williams
+## MCL1 Set
+## Software Used
+1. Maestro Protein Prep v.2024-02ß
+
```

Comment:
Could you add additional details on the steps you took to prepare the protein and the ligands?